papas/reconstruction/src/PFBlock.cpp

```diff
@@ -68,12 +68,20 @@ std::string PFBlock::shortName() const {
   return out.str();
 }
 
-const Edge& PFBlock::findEdge(Edge::EdgeKey key) const {
+const Edge& PFBlock::edge(Edge::EdgeKey key) const {
   auto edge = m_edges.find(key);
   if (edge == m_edges.end()) throw std::range_error("Edge not found");
   return edge->second;
 }
 
+const Edge& PFBlock::edge(Identifier id1, Identifier id2) const {
+  /// Find the edge corresponding to e1 e2
```

Comment:
put that as a doxygen docstring in the .h and remove. 
no need to get into details, just say that the order of the ids does not matter.